### DISCUSSION TITLE
Live Preview (GUI improvements): Tabs should stack vertically as accordion

### DISCUSSION THREAD

**Discussion by AlexFUNBIT** [craftcms/cms#10217] [Ideas]
# Hi all 😄 

## I love the live preview feature (and so does my clients).

**Here is my humble suggestion for better user experience!  
Tabs should stack (like accordion) or (as a quick fix), use the overflow-x solution done on mobile devices.**

### 1) Use mobile layout in live preview. 

![overflow-x](https://user-images.githubusercontent.com/10828933/144939252-bb3a761e-d6f1-40c9-be92-99839cc331a9.png)

Pic: You have already made this layout for mobile users. It will do great in live preview as a fast solution. 

### 2) Tabs should be - accordion when content flows vertically

![tabs_stacked](https://user-images.githubusercontent.com/10828933/144942887-49b3824b-de83-47c1-9a48-14e1c635f4f5.png)

Pic: Tabs stacked as accordion. Craft CMS design language.

![mocup](https://user-images.githubusercontent.com/10828933/144943666-ed419fbd-babc-49aa-a08c-093850f8db87.png)

Pic: Mocup, [High-Res Image](https://i.imgur.com/KJj6dM9.png)

Behave like show in [this codepen](https://codepen.io/imshashankdogra/pen/PgXbvx).

**Comment by brandonkelly** [comment]
Related: #8896
